azure-kit-starter/src/main/java/com/vaadin/azure/starter/ui/VersionNotificator.java

```diff
@@ -0,0 +1,53 @@
+package com.vaadin.azure.starter.ui;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.vaadin.flow.component.Component;
+import com.vaadin.flow.component.ComponentEvent;
+import com.vaadin.flow.component.ComponentEventListener;
+import com.vaadin.flow.component.DomEvent;
+import com.vaadin.flow.component.Tag;
+import com.vaadin.flow.component.dependency.JsModule;
+
+/**
+ * The version notificator component is used when there is a new version and the
+ * update version cookie is set.
+ */
+@Tag("version-notificator")
+@JsModule("./components/version-notificator.ts")
+public class VersionNotificator extends Component {
+    private static final long serialVersionUID = 1L;
+
+    VersionNotificator(String current, String update) {
+        getElement().setProperty("currentVersion", current);
+        getElement().setProperty("updateVersion", update);
+    }
+
+    /**
+     * Adds a listener to listen for the switch version events.
+     */
+    public void addSwitchVersionEvent(
```

Comment:
Shall we name it `addSwitchVersionListener`?

Reply:
Renamed it to `addSwitchVersionListener`.